lib/terraforming/resource/efs.rb

```diff
@@ -0,0 +1,56 @@
+module Terraforming
+  module Resource
+    class EFSFileSystem
+      include Terraforming::Util
+
+      def self.tf(client: Aws::EFS::Client.new)
+        self.new(client).tf
+      end
+
+      def self.tfstate(client: Aws::EFS::Client.new)
+        self.new(client).tfstate
+      end
+
+      def initialize(client)
+        @client = client
+      end
+
+      def tf
+        apply_template(@client, "tf/elastic_filesystem")
+      end
+
+      def tfstate
+        file_systems.inject({}) do |resources, efs|
+          attributes = {
+            "creation_token" => efs.creation_token,
+            "id" => efs.file_system_id,
+            "performance_mode" => efs.performance_mode,
+            "tags.%" => "1",
+            "tags.Name" => efs.name,
+          }
+
+          resources[format('%s', efs.file_system_id)] = {
```

Comment:
Looks like I did in fact miss the aws_efs_file_system prefix in the resource name. Also changed to interpolation (a style that I personally hate, btw, but you're the owner, so... :))